SimplenoteIntents/SimplenoteIntents.entitlements

```diff
@@ -4,7 +4,7 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.com.codality.NotationalFlow.Alpha</string>
+		<string>group.com.codality.NotationalFlow</string>
```

Comment:
This and the other changes to the `.entitlements` were only to keep them consistent with the widget's ones.

These are just file names at the end of day, they could have stayed the way they were, but consistency is useful to lower the cognitive load of navigating the project.